apps/publisher/themes/default/partials/lifecycle-state-duration.hbs

```diff
@@ -0,0 +1,23 @@
+<!--
```

Comment:
Shall we have this as Handlebars comments, other wise these will appear in page.